src/pages/roadmap.md

```diff
@@ -2,46 +2,50 @@
 hide_table_of_contents: true
 ---
 
-# Roadmap for XMTP
+# XMTP roadmap
 
-Timelines are estimates and subject to change. For information about delivered features and fixes, see the [Changelog](/docs/changelog).
+The features and timelines listed below are subject to change. For information about recently delivered features and fixes, see the [Changelog](https://xmtp.org/docs/changelog).
 
-## Release cycle
+## 2023
 
-We typically follow this release cycle:
+### Q4
 
-- Alpha: Super early peek into the feature being built. Not ready for use in production apps.
-- Beta: Opportunity to test, validate, and provide feedback. Not ready for use in production apps.
-- Production: Ready for use in production apps.
+- Group chat prototype powered by Messaging Layer Security (MLS)
+- Shared allow/deny lists for iOS/Android SDKs
+- Publish a decentralization overview covering contacts, invites, inboxes, and conversations
+- Contact directory prototype (a smart contract that allows users to create, maintain, and share a public DID contact bundle) available in alpha
 
-## Features roadmap
+## 2024
 
-### 2023 Q3
+### Q1
 
-- Onchain transaction reference content type
-- React SDK in Production
-- React Native SDK in Production (currently in Beta)
-- Android SDK in Production (currently in Beta)
-- XMTP v3 in Alpha
+- Group chat beta available in iOS, Android, and React Native SDKs
+- Initial support for push notifications, message history backups/portable inboxes, and smart contract wallet accounts
+- Contact directory available in production
```

Comment:
Does this mean the contact directory is available on mainnet for ethereum? Or does production mean it's piped all the way through the protocol and being used in the SDK. @jac18281828